_includes/fix_analytics_signup_form.html

```diff
@@ -351,6 +358,26 @@ <h2>
             return result;
         }
 
+        function updatePrice() {
+            const domainList = document.querySelectorAll("[data-domain-field]");
+            const dkkPriceElement = document.querySelector("[data-price-dkk]");
+            const eurPriceElement = document.querySelector("[data-price-eur]");
+            let totalPriceDkk = 0;
+            let totalPriceEur = 0;
+            domainList.forEach(function(input) {
+                if (input.value) {
+                    totalPriceDkk += 37;
+                    totalPriceEur += 5;
+                }
+            });
```

Comment:
I can't decide if I think it's nicer to do this with a Math.sum or something similar... Or maybe just maths like this?

```js
const domainCount = domainList.filter(function(input) {
  return input.value;
});
const totalPriceDkk = domainCount * 37;
const totalPriceEur = domainCount * 5;
```

Reply:
And honestly now you don't need to calculate these ahead of time, they can just be calculated on lines 374+377 where the values are set.

Reply:
Which means we don't calculate both values, seeing as we only need one of them in the view (either DKK or EUR depending on language selected).

Reply:
Can't filter on a nodeList, so this is not possible in an easy way.